stdlib/crypto/src/main/java/org/ballerinalang/stdlib/crypto/nativeimpl/Hmac.java

```diff
@@ -1,20 +1,22 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why do we change the licence here? These files were created back in 2017. This year change might occur some conflicts with @ Since annotation as well

Reply:
The IDE seems to be changing/formatting the copyright when moving the files